app/controllers/providers/check_benefits_controller.rb

```diff
@@ -1,11 +1,24 @@
 module Providers
   class CheckBenefitsController < ProviderBaseController
     def index
+      return set_negative_result_and_go_forward if know_issue_prevents_benefit_check?
```

Comment:
is this supposed to be ```know_issue``` as in 'we know this issue exists' or should it be ```known_issue``` to match up with the 'skipped:known_issue' result ?